Doc/library/math.rst

```diff
@@ -247,6 +249,30 @@ Floating point arithmetic
    .. versionadded:: 3.13
 
 
+.. function:: fmax(x, y, /)
+
+   Get the larger of two floating-point values, treating NaNs as missing data.
+
+   When both operands are (signed) NaNs or zeroes, return ``nan`` and ``0``
+   respectively and the sign of the result is implementation-defined, that
+   is, :func:`!fmax` is not required to be sensitive to the sign of such
+   operands (see ISO C11, Annexes F.10.0.3 and F.10.9.2).
+
+   .. versionadded:: next
+
+
+.. function:: fmin(x, y, /)
+
+   Get the smaller of two floating-point values, treating NaNs as missing data.
+
+   When both operands are (signed) NaNs or zeroes, return ``nan`` and ``0``
+   respectively and the sign of the result is implementation-defined, that
+   is, :func:`!fmin` is not required to be sensitive to the sign of such
+   operands (see ISO C11, Annexes F.10.0.3 and F.10.9.3).
```

Comment:
```suggestion
   operands (see Annex F of the C11 standard, §F.10.0.3 and §F.10.9.2).
```

Reply:
I'll use F.10.9.3 instead of your original F.10.9.2 which is for fmax() (though 10.9.3 says to see 10.9.2 but just in case).